src/main/java/com/redhat/insights/agent/AgentMain.java

```diff
@@ -181,7 +184,7 @@ private void start() {
 
   private Supplier<InsightsHttpClient> getInsightsClientSupplier() {
     final Supplier<InsightsHttpClient> out;
-    if (configuration.isDebug() || configuration.isFileOnly()) {
```

Comment:
This was an or before, meaning it would log even if `isFileOnly()` was false but `isDebug()` was true. Now it will not log if debug logging is turned on but `isFileOnly()` is false.

Reply:
Yes, this change (don't force fileOnly if debug is true) is intentional and was specifically requested by QE to debug network connectivity issues.